CHANGELOG.md

```diff
@@ -7,4 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/metamask/delegation-toolkit/
+## [7.0.0]
```

Comment:
We don't typically keep a changelog for the entire monorepo, just packages. You don't need to update this file per release, in fact you should be able to remove it.